e2e-flap.sh

```
@@ -0,0 +1,789 @@
+COMPOSE_FILE="docker-compose.e2e-flap.yaml"
+E2E_SECRETS_DIR=".e2e-secrets"
+TS_CONTAINER="e2e-flap-tailscale"
+CLIENT_CONTAINER="e2e-flap-client"
+DOCKTAIL_CONTAINER="e2e-flap-docktail"
+API_BASE="https://api.tailscale.com/api/v2"
+API_TAILNET="${TS_TAILNET:--}"
+
+MAX_WAIT=120
+BASELINE_BUDGET=150
+FLAP_BUDGET=90
+DRAIN_OBSERVE=90
```

Shellcheck (0.11.0)

[warning] DRAIN_OBSERVE appears unused. Verify use (or export if used externally).

(SC2034)